### PR TITLE
OSDOCS-11986 OCP 4.15.32 Release Notes (Sept. 18 release)

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.15

Issue: [OSDOCS-11986](https://issues.redhat.com/browse/OSDOCS-11986)

Link to[ docs preview:](https://81994--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes.html#ocp-4-15-32_release-notes)

QE review: Not needed
- [ ] QE has approved this change.

Additional information: Errata links will not work until Sept. 18th.

### CODE REVIEW

**Review by bergerhoffer — COMMENTED** [review]
Just a few things!

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -2740,6 +2740,38 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, the installation program populated the `network.devices`, `template` and `workspace` fields in the `spec.template.spec.providerSpec.value` section of the {vmw-full} control plane machine set custom resource (CR).
+These fields should be set in the {vmw-short} failure domain, and the installation program populating them caused unintended behaviors. Updating these fields did not trigger an update to the control plane machines, and these fields were cleared when the control plane machine set was deleted.
+
+With this release, the installation program is updated to no longer populate values that are included in the failure domain configuration. If these values are not defined in a failure domain configuration, for instance on a cluster that is updated to {product-title} {product-version} from an earlier version, the values defined by the installation program are used. (link:https://issues.redhat.com/browse/OCPBUGS-37064[*OCPBUGS-37064*])
```
[the remark this answers is not in the document]
Comment (cbippley) [reply release_notes/ocp-4-15-release-notes.adoc:2767]: Those two are the same bug, I'll try to make that more clear.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -2740,6 +2740,38 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+----
+$ oc adm release info 4.15.32 --pullspecs
+----
+
+[id="ocp-4-15-32-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the Cloud Credential Operator (CCO) would produce an error when starting or restarting when there were a large number of secrets in the cluster fetched at once. With this release, CCO fetches the secrets in batches of 100 and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-41235[*OCPBUGS-41235*])
+
+* Previously, the installation program populated the `network.devices`, `template` and `workspace` fields in the `spec.template.spec.providerSpec.value` section of the {vmw-full} control plane machine set custom resource (CR).
```
[the remark this answers is not in the document]
Comment (cbippley) [reply release_notes/ocp-4-15-release-notes.adoc:2764]: I'm adding a + to connect it to the next paragraph which is the same bug.

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply release_notes/ocp-4-15-release-notes.adoc:2767]: Oh gotcha. You should be able to just put a "+" in the line between them, and then it'll indent it properly with it

**Pull request merged**